.travis.yml

```diff
@@ -1,6 +1,12 @@
 os:
   - linux
   - osx
+node_js:
+  - 0.10
+  - 0.12
+  - 4
+  - 6
```

Comment:
for next releases we should just bother with 6,8,10 IMHO and forget the old versions

Reply:
I'll clean up this PR tomorrow, and remove unsupported LTS releases of node.